src/lib/Lib/Filters.hpp

```diff
@@ -0,0 +1,98 @@
+//
+// This is a derivative work. originally part of the LLVM Project.
+// Licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// Copyright (c) 2023 Krystian Stasiowski (sdkrystian@gmail.com)
+//
+// Official repository: https://github.com/cppalliance/mrdox
+//
+
+#ifndef MRDOX_LIB_FILTERS_HPP
+#define MRDOX_LIB_FILTERS_HPP
+
+#include <mrdox/Platform.hpp>
+#include <span>
+#include <string>
+#include <string_view>
+#include <vector>
+
+namespace clang {
+namespace mrdox {
```

Comment:
NIT: why not `namespace clang::mrdox {` ?

Reply:
@fpelliccioni The project convention is to declare each namespace individually

Reply:
👍